meetings/2021-09-08.md

```diff
@@ -0,0 +1,140 @@
+but after that they should possibly last forever.
+
+**Barath** I agree, that well chosen modes will last for long time. Even substandard modes serve in existing systems.
+
+**Justin** I think delete is good example, it can be set on the level of container and on individual resources. I don't see argument that there could be delete without implicit update of containment triples.
+Argument that something else gets changed doesn't sound for me like recipe for good authorization system.
+
+**Pavlik** I think create and delete without allowing updating client managed triples in the
+container is a good scenario that is needed to be addressed.
+
+**Sarven** If you look at access modes in a vacum there may be all kind of modes that can be discussed.
+If we have protocol which has hierarchical containment ...
+Protocol usually starts with read and write. When you interacting - making a request against specific
+type of resource eg. container. It is processing operation, where protocol defines what happens.
+In WAC in order to delete something, one needs to have write on the resource and write on container
+containing it.
+
+Another thing that seems to be conflated is resource level and content level.
+
+[isssue 121: Towards an authorization framework for the Solid ecosystem](https://github.com/solid/authorization-panel/issues/121)
+We still haven't solved that issue, it can help us clarify where each use case fits in.
+I think we are ripping this issue apart instead of tackling it directly.
+I see operations in the protocol territory. I don't think solution needs to demand some particular
+access modes.
+
+If we look at PUT, from HTTP perspective you either create issue first or in subsequent requests
+you replace / update it.
+
+**Henry** I've been looking for some math that could help put some clarity in the problem. We are dealing with a prtocol between a client and a server, so that for example limits what we need to cover: we don't need to cover that the server has writes to make the changes.
+We sould assume that server has rights to do what it needs to do. I was mentioning Lenses: these are a generalisation of what we know in Java Beans with getters and setters. Having getters and setters one can do CRUD.
+I'm looking for papers on those topics. One thing that stands out clearly is that `Control` is not part of the space. It seems to be a
+different concept. We could remove control and focus how CRUD can be modeled.
+I'm trying to understand why one would need access to the container. I don't think one needs write
+on the container to delete resource. We can see if there are any potential security holes by building up an example with and without write. 
+
+**Justin** I'm able to delete comment in a container of comments. I should be able to delete mine but
+not anyone's else. Delete on the container, meaning that it applies to all the comment would allow
+deleting all the comments.
```

Comment:
```suggestion
**Justin** I'm able to `delete` a comment in a container of comments. I should be able to delete mine but
not anyone's else. `Delete` on the container, meaning that it applies to all the comments would allow
deleting all the comments.
```